projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -911,9 +924,7 @@ public function process_request( $post, array $post_data ) {
 			'https://twitter.com/intent/tweet'
 		);
 
-		// Redirect to Twitter
-		wp_redirect( $twitter_url );
-		die();
+		parent::redirect_request( $twitter_url );
```

Comment:
The use of `parent::` rather than `$this->` seems weird to me, but it works.

Reply:
I mostly followed the pattern in place for that class on this one.